roles/common/meta/main.yml

```diff
@@ -0,0 +1,3 @@
+---
```

Comment:
is this a new file, or a changed name file?

Reply:
i think the issue is this file needs to be moved/ edited in separate commits: https://stackoverflow.com/questions/433111/how-to-make-git-mark-a-deleted-and-a-new-file-as-a-file-move/433114

Reply:
It is moved - from here: https://github.com/confluentinc/cp-ansible/blob/6.2.0-post/roles/confluent.common/meta/main.yml

Reply:
It is marked as moved in the commit: https://github.com/confluentinc/cp-ansible/pull/698/commits/ed2b43a9f5223d8409fa81975a68de878e376370

I think you have to look at the individual commits, and not the PR diff.

Reply:
ok great! thanks for confirming